src/app/layout/statistics/statistics.component.html

```diff
@@ -5,7 +5,9 @@
       <ng-container i18n="@@public-layout.amount_ids">
         ORCID iDs and counting.
       </ng-container>
-      <a i18n="@@ngOrcid.seeMoreStatics" href="/statistics"
+      <a
+        i18n="@@ngOrcid.seeMoreStatics"
```

Comment:
@amontenegro Could you add the missing attributes  `target="_blank"`
and `rel="noopener noreferrer"` since now statistics is in the info site